presto-docs/src/main/sphinx/functions/hive.rst

```diff
@@ -0,0 +1,49 @@
+==============================
+Hive Functions
+==============================
+
+It is a way of reuse of the hive functions in presto. All of the hive
+function classes exist in presto or external storage. We encode / decode
+the parameters to transform the type between presto and hive, and then
+wrap the hive function as the presto function.
+
+The Hive Functions has 3 components:
+
+* Hive Scalar Function
+* Hive Aggregation Function
+* Hive Window Function
+
+We support hive built-in scalar function now.
+
+Hive Scalar Functions
+-----------------
+
+.. note::
+
+    Hive has two different interfaces for writing Hive UDF.
+    One is the basic UDF interface (deprecated) and the other
+    is the GenericUDF interface.
+
+    We support hive built-in GenericUDf now.
```

Comment:
```suggestion
    Hive has two different interfaces for writing Hive UDF.
    One is the basic UDF interface (deprecated) and the other
    is the GenericUDF interface.

    We currently only support the Hive built-in GenericUDF.
```

Reply:
That's very nice of you!
I have modified the doc according to your comments and add some describes about the hive udf.